File: DomZaStaraLicaApi/Endpoints/Nutricionista/DodajNutricionistu/DodajNutricionistuRequest.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace DomZaStaraLicaApi.Endpoints.Nutricionista.DodajNutricionistu
{
    public class DodajNutricionistuRequest
    {
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public string NutricionistickiCentar { get; set; }
        public string OblastNutricionizma { get; set; }
    }
}

File: frontend/src/app/nutricionista/dodajNutricionistuRequest.ts
export interface DodajNutricionistuRequest {
    imePrezime: string;
    jmbg: string;
    datumRodjenja: Date;
    datumZaposlenja: Date;
    nalogId: number| null;
    poslovnaPozicijaId: number;
    nutricionistickiCentar: string;
    oblastNutricionizma: string;
}
